extensions/amp-story/1.0/bookend/amp-story-bookend.js

```diff
@@ -91,6 +93,14 @@ const REPLAY_ICON_TEMPLATE = {
 /** @const {string} */
 const TAG = 'amp-story-bookend';
 
+/** @enum {string} */
+const CLICKABLE_COMPONENT_TYPES = {
+  CTA_LINK: 'i-amphtml-story-bookend-cta-link',
+  LANDSCAPE: 'i-amphtml-story-bookend-landscape',
+  PORTRAIT: 'i-amphtml-story-bookend-portrait',
+  SMALL: 'i-amphtml-story-bookend-article',
+};
```

Comment:
This is very fragile as the classeNames we actually set aren't from this ENUM. I think it's worth trying to see how we can change the way we set classNames/attributes to make it re-usable, instead of adding a new standalone ENUM.
As a bonus, depending on how you implement this, you might be able to get rid of the `getClickedComponentIndex_` and `getClickedComponentType_` methods.